include/mgmt/mgmt_fmfu.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file mgmt_fmfu.h
+ * @defgroup mgmt_fmfu Management Full Modem Firmware Upgrade 
```

Comment:
Is `management` a known term? Should we state that it is an SMP Manager? I'm not sure about how to phrase it, maybe we could look at existing mgmt header files.